examples/relay-angular-chat/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "relay-angular-chat",
+  "version": "0.1.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "watch": "ng build --watch --configuration development",
+    "test": "ng test"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "~13.2.0",
+    "@angular/common": "~13.2.0",
+    "@angular/compiler": "~13.2.0",
+    "@angular/core": "~13.2.0",
+    "@angular/forms": "~13.2.0",
+    "@angular/platform-browser": "~13.2.0",
+    "@angular/platform-browser-dynamic": "~13.2.0",
+    "@angular/router": "~13.2.0",
+    "crypto-browserify": "^3.12.0",
+    "js-waku": "file:../../build/esm",
+    "protons": "^2.0.3",
```

Comment:
We should move away from protons: https://github.com/status-im/js-waku/issues/335

I think it's ok to merge this PR with protons, it's more of an FYI.